### PR TITLE
Sergkanz/rddcleanup

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Allowing to override target and aligning names with Application Analytics schema. Also removing some obsolete properties

### CODE REVIEW

File: Test/CoreSDK.Test/Shared/DataContracts/DependencyTelemetryTest.cs [hunk 1]
```diff
@@ -88,58 +81,48 @@ public void RemoteDependencyTelemetrySerializeEmptyCommandNameToJson()
...
+#pragma warning disable 618
+            expected.DependencyKind = "Http";
 
-        [TestMethod]
-        public void DependencyKindIsSetIfValueCanBeParsedFromEnum()
-        {
-            DependencyTelemetry expected = new DependencyTelemetry();
-            expected.DependencyKind = BondDependencyKind.Http.ToString();
+            Assert.Equal("Http", expected.DependencyKind);
+            Assert.Equal("Http", expected.DependencyTypeName);
```
Comment (dnduffy) [inline Test/CoreSDK.Test/Shared/DataContracts/DependencyTelemetryTest.cs]: nitpick: Could make a variable or const for "Http".

File: Test/CoreSDK.Test/Shared/TelemetryClientTest.cs [hunk 2]
```diff
@@ -841,7 +841,7 @@ public void SendEventToValidateEndpoint()
 
             HttpClient client = new HttpClient();
             var result = client.PostAsync(
-                "https://dc.services.visualstudio.com/v2/validate",
+                "https://eus-breeze-in.cloudapp.net/v2/validate",
```
Comment (dnduffy) [inline Test/CoreSDK.Test/Shared/TelemetryClientTest.cs]: Just curious, why this particular endpoint instance?

Comment (SergeyKanzhelev) [inline Test/CoreSDK.Test/Shared/TelemetryClientTest.cs]: Oh, forgot to mention. The updated schema is not deployed yet. Thus PR is for separate branch, not develop. I'd like to accumulate all the changes and then push together in develop

**Comment by dnduffy** [comment]
So just to confirm, CommandName became Data and Data became InternalData?
:shipit:

File: Test/CoreSDK.Test/Shared/TelemetryClientTest.cs (hunk 2 above)
Comment (dnduffy) [inline Test/CoreSDK.Test/Shared/TelemetryClientTest.cs]: That's fine, are you saying this endpoint could change in the final merge to develop?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 This changelog will be used to generate documentation on [release notes page](http://azure.microsoft.com/en-us/documentation/articles/app-insights-release-notes-dotnet/).
 
+- Remote dependency type was updated to reflect the latest developments in Applicaiton Insights Applicaiton Map feature. You can set a new field - `Target`. `CommandName` was renamed to `Data` for consistancy with the Application Analytics schema. `DependencyKind` will never be send any more and will not be set to "Other" by default. Also there are two more constructors for `RemoteDependencyTelemetry` item.
```
Comment (abaranch) [inline CHANGELOG.md]: >Remote [](start = 2, length = 6)

This should go under Version 2.2.0-beta2

Comment (abaranch) [inline CHANGELOG.md]: >Remote [](start = 2, length = 6)

It is not "remote"

**Comment by abaranch** [comment]
:shipit:

File: Test/CoreSDK.Test/Shared/TelemetryClientTest.cs (hunk 2 above)
Comment (SergeyKanzhelev) [inline Test/CoreSDK.Test/Shared/TelemetryClientTest.cs]: nope. I'll revert back before merge to `develop`

File: CHANGELOG.md (hunk 3 above)
Comment (SergeyKanzhelev) [inline CHANGELOG.md]: I'll fix it

**Pull request merged**